modules/mod_ginger_base/tests/mod_ginger_base_rest_tests.erl

```diff
@@ -0,0 +1,20 @@
+-module(mod_ginger_base_rest_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+-include("zotonic.hrl").
+
+-record(state, {mode}).
+
+resource_exists_test_() ->
+    [
+     fun () ->
```

Comment:
No space between `fun` and parens. While not explicitly mentioned by Inaka’s guidelines, let’s stick to [their practice](https://github.com/inaka/erlang_guidelines/blob/e2b67c0d48c7348185e6584f9636c12a777277ef/src/import.erl#L9).

Reply:
Hmm, that doesn't make sense to me; the form is `fun [Name]...` (see http://erlang.org/doc/reference_manual/expressions.html#id84308) where `Name` is optional, therefore I prefer to do have a space between `fun` and `()`.

Reply:
Also note that the Erlang reference manual *does* a space.

Reply:
Yep, but we follow Inaka’s guidelines.